kde-apps/libkface/libkface-15.08.2-r1.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit kde4-base
+
+DESCRIPTION="Qt/C++ wrapper around LibFace to perform face recognition and detection"
+HOMEPAGE="http://api.kde.org/4.x-api/kdegraphics-apidocs/libs/libkface/libkface/html/index.html"
```

Comment:
That must be new, maybe temporary issue because of master branch moving to frameworks